### PR TITLE
Adding recipe for pyemojify.

### PR DESCRIPTION
**Pull request by hellothisisMatt**
New recipe for pyemojify

### CODE REVIEW

File: recipes/pyemojify/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    - python
+    - pytest
+
+test:
+  imports:
+    - pyemojify
+
+
+about:
+  home: https://github.com/lord63/pyemojifyl
```
Comment (CJ-Wright) [inline recipes/pyemojify/meta.yaml]: This page doesn't seem to exist.

File: recipes/pyemojify/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5sum }}
+
+build:
+  number: 0
+
+requirements:
+  build:
```
Comment (CJ-Wright) [inline recipes/pyemojify/meta.yaml]: I think you need python here.

File: recipes/pyemojify/build.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python setup.py install --single-version-externally-managed --record record.txt
```
Comment (CJ-Wright) [inline recipes/pyemojify/build.sh]: you might consider putting this under the build section as:
```
build:
   script: python setup.py install --single-version-externally-managed --record record.txt
```

File: recipes/pyemojify/meta.yaml (hunk 1 above)
Comment (CJ-Wright) [reply recipes/pyemojify/meta.yaml]: Or at least I can't find it.

File: recipes/pyemojify/build.sh (hunk 3 above)
Comment (hellothisisMatt) [reply recipes/pyemojify/build.sh]: Does this mean I wouldn't need a build.sh file then?

Comment (CJ-Wright) [reply recipes/pyemojify/build.sh]: Yes, you can look at the example file for an example of this.

File: recipes/pyemojify/meta.yaml (hunk 2 above)
Comment (CJ-Wright) [inline recipes/pyemojify/meta.yaml]: If you aren't using [selectors](https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#preprocessing-selectors) you can use [noarch](https://conda-forge.org/docs/meta.html#building-noarch-packages).

**Comment by hellothisisMatt** [comment]
Thanks for the review @CJ-Wright. I hope the changes I've made are correct :)

File: recipes/pyemojify/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  md5: {{ md5sum }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
```
Comment (CJ-Wright) [inline recipes/pyemojify/meta.yaml]: This needs `setuptools`

Comment (hellothisisMatt) [reply recipes/pyemojify/meta.yaml]: Added

File: recipes/pyemojify/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+test:
+  imports:
+    - pyemojify
+
+
+about:
+  home: https://github.com/lord63/pyemojify
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```
[the remark this answers is not in the document]
Comment (hellothisisMatt) [reply recipes/pyemojify/meta.yaml]: I'd suppose not, I'll remove the redundancy.

[the remark this answers is not in the document]
Comment (CJ-Wright) [reply recipes/pyemojify/meta.yaml]: Just remove one redundancy, not both. This recipe needs a `license_file`.

**Comment by CJ-Wright** [comment]
@conda-forge-linter, please lint.

File: recipes/pyemojify/meta.yaml (hunk 5 above)
[the remark this answers is not in the document]
Comment (hellothisisMatt) [reply recipes/pyemojify/meta.yaml]: So add the license_file back to the recipe, and remove the license that comes in the tarball?

[the remark this answers is not in the document]
Comment (CJ-Wright) [reply recipes/pyemojify/meta.yaml]: Yes, we'll pull it from the tarball itself when we download it to install the package.

**Comment by scopatz** [comment]
LGTM, feel free to merge

**Pull request merged**